am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -176,21 +189,26 @@ public FilterResourceResponse filterResource(@NotBlank String userId,
             attributePermissions = roleBasedAccess.stream().collect(getMapCollector());
 
         } else {
-            attributePermissions = explicitAccess.stream().collect(getMapCollector());
+            attributePermissions = permissionsService.merge(permissionsForRelationships);
         }
 
         JsonNode filteredJson = filterService.filterJson(resource.getResourceJson(), attributePermissions);
 
+        Set<String> relationships = explicitAccess.stream()
```

Comment:
what should relationships look like when there is only role based access records?